src/components/App.tsx

```diff
@@ -331,6 +331,23 @@ export class App extends React.Component<any, AppState> {
       }
     }
 
+    // rerender text elements on font load to fix #637
+    try {
+      await Promise.race([
+        document.fonts?.ready?.then(() => {
```

Comment:
Is the second `?` necessary? I think it just resolves to `undefined`.
```suggestion
        document.fonts?.ready.then(() => {
```

Reply:
In case `fonts` is defined, by `ready` isn't. I have clue if that can happen though, so it's defensive programming on my part.